tribits/ci_support/cdash_analyze_and_report.py

```diff
@@ -222,12 +222,26 @@ def injectCmndLineOptionsInParser(clp, gitoliteRootDefault=""):
       " file names.",
     clp )
 
+  addOptionParserChoiceOption(
```

Comment:
Probably outside the scope of this review, but this function name seems to have some redundancy in it with `Option` showing up twice.

Reply:
Well, you are adding a "choice option" to an "OptionParser" object.  Since the default function is `add_option()`, this seemed like the best name.

Reply:
NOTE: With static typing in C++, you would just call this non-member function `addChoiceOption()` since the first argument would be of type `OptionParser` and that would disambiguate this function from other functions of the same name that are meant to operate on other objects.  But since Python objects are not statically typed, you need to add the class name in the non-member function name to avoid possible conflicts and avoid confusion.  (The pros and cons of dynamically typed vs. statically typed languages.)